README.md

```diff
@@ -105,6 +105,14 @@ model = LLM("TinyLlama-1.1B-Chat-v1.0-INT8")
 output = model.generate("My name is")
 ```
 
+## New Features
```

Comment:
Can we put this at the top?
And a little bit "louder" - (some bigger text, a rocket emoji, etc)

Reply:
Updated, does this look good?